xinference/client/tests/test_client_with_auth.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 import pytest
+from openai import AuthenticationError, OpenAI, PermissionDeniedError
```

Comment:
Cannot import here. Must be imported inside the function. Since not everyone install xinference with dev option.